packages/server/controllers/auth.js

```diff
@@ -240,15 +240,15 @@ async function sendPasswordResetEmail(address, passwordResetToken) {
   const link = `http://localhost:5050/reset-password?token=${passwordResetToken}`;
```

Comment:
We can't hardcode the link to localhost because this would work just for testing only. We'll have to use an environment variable for the base URL that changes depending on whether it's the development environment or production.

I'll send more details in Discord.